### ISSUE TITLE
PP downloadable as apt package

### ISSUE THREAD

**Issue by philippnagel** [buchen/portfolio#1958]
Hi,

I prefer installing software via apt (Ubuntu et al.). Hence I'd like to know whether this has already been explored for PP?

I am more than happy to work on this, unless the project maintainers are strictly against it.

Best regards,
Phil

**Comment by chirlu** [comment]
I don’t think anyone would be against it. However, you may be underestimating the task, considering the many (often Eclipse-related) dependencies that PP has.

**Comment by metal450** [comment]
Agreed - would be super convenient to have this checked/updated via the package manager :)
